core/worker.go

```diff
@@ -92,8 +97,14 @@ func (w *WorkerManager) parseCommand(cmd string) *Command {
 func (w *WorkerManager) close() {
 	w.wg.Wait()
 	close(w.semaphore)
-	// Workermanager is responsible for logging, hence we run the close routine
+}
+
+// closeStdout prints stats message if it is enabled
+// and closes stdout channel.
+func (w *WorkerManager) closeStdout(force bool) {
+	// WorkerManager is responsible for logging, hence we run the close routine
 	// here.
+	stats.Print(force)
```

Comment:
why can't we call this function in WorkerManager's close() method?

Reply:
Run() mode(file) prints stats message as a default while RunCmd() does not. That is why a flag like `force` is required to preserve standard behavior. I did not want to pass a boolean parameter to close() method since it was confusing and only required for printing stats. `close()` sounds like a WorkerManager functionality to me. What do you think? Should we change the signature of close() to close(bool)?